python/voici/pyproject.toml

```diff
@@ -44,7 +44,7 @@ file = "LICENSE"
 
 [project.optional-dependencies]
 dev = [
-    "ruff==0.0.258",
+    "ruff==0.4.5",
```

Comment:
Maybe we can keep the same pin as in https://github.com/voila-dashboards/voici/blob/dd123e04a7a99a63a606667e6389a09e49f2d9f0/python/voici-core/pyproject.toml#L47 ?